src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -98,7 +100,23 @@ public void ClientAsyncAuthenticate_MismatchProtocols_Fails(SslProtocols server,
             Assert.Throws(expected, () => ClientAsyncSslHelper(server, client));
         }
 
+        [Theory]
+        [MemberData("ProtocolMismatchDataSsl2Specific")]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
@stephentoub The notes say that OpenSsl doesn't allow SSL2.

@shrutigarg Please add a Unix test which verifies whatever failure is expected for SSL2 on Unix.